### PR TITLE
150 - Show article source in MDX

### PR DESCRIPTION
**Pull request by TomaszMarczak**
Changed GA event to "article_source_clicked" for consistency
Known bug: Article source not showing properly on first article: "Loops in JavaScript and their performance".
image

### CODE REVIEW

File: src/components/article/Article.tsx [hunk 1]
```diff
@@ -110,9 +116,18 @@ export default function ({
     previous,
     translations,
     lang,
+    rawBody,
   } = article
 
   const t = translationObject[lang]
+  const { track } = useCustomGAEvent()
+  const articleSourceModal = useModal()
+  const articleSource = rawBody.replace(/^(---\s*\n[\s\S]*?\n?)?---\s*\n/, "")
```
Comment (polubis) [inline src/components/article/Article.tsx:125]: This must be performed ArticleSource component to avoid doing that in Articles

**Review by polubis — CHANGES_REQUESTED** [review]
Great work one bug to fix and one improvement required

**Review by polubis — APPROVED** [review]
I remember that there was a problem with PL articles, yup?  We can solve it on call but code itself is great, good work mate!

**Comment by TomaszMarczak** [comment]
Issue was with 2 articles, where modal was broken: "Use Fetch Hook" and "Improving SEO with Ahrefs"
PL articles are working properly.

File: src/components/article/TranslatedArticlePage.tsx [hunk 2]
```diff
@@ -180,9 +201,20 @@ export default function ({
                 {getDistanceLabel(modifiedAt, t.dates.updated)}
               </Badge>
             </Dates>
+            <BottomNavigation>
+              <Button
+                style={{ marginRight: "auto" }}
```
Comment (polubis) [inline src/components/article/TranslatedArticlePage.tsx:206]: Don't use inline styles. Add it to BottomNavigation styled. Mixing inline with not inline approach can create problem with maintenance later.

**Review by polubis — APPROVED** [review]
LGTM - > please add small fix and I'll merge that. Looks good during testing. Good job

**Pull request merged**